lib/pbench/agent/results.py

```diff
@@ -1,98 +1,118 @@
+from configparser import ConfigParser
 import datetime
 import errno
+from logging import Logger
 import os
+from pathlib import Path
 import requests
-import sys
 import tarfile
-
-from configparser import ConfigParser
-from pathlib import Path
-
+from typing import IO, Iterator
```

Comment:
As was mentioned in PR #2214, let's not change the way we are handling imports until we have had some discussion.  We have been keeping `import` and `from` separate in each of the 3 sections defined by PEP 8 (standard, 3rd party, and local).

Reply:
I've reordered the imports per my IDE's preference which seems to match PEP 8.  The differences between the previous code and my proposed code now show additions, removals, and very few relocations (mostly `responses`, which is 3rd-party, not standard).